app/graphql/mutations/create_battle.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateBattle < BaseMutation
+    null true
+    argument :character1, Int
+    argument :character2, Int
+
+    type Types::BattleType
+
+    def resolve(character1:, character2:)
+      character1 = Character.find_by(id: character1)
+      character2 = Character.find_by(id: character2)
```

Comment:
This looks awesome!  If we need to adjust this logic we can come look at this later for FE stuff or if stuff with chuck needs to be changed.